packages/template-retail-react-app/app/static/translations/compiled/en-US.json

```diff
@@ -767,6 +767,16 @@
       "value": "Salesforce or its affiliates. All rights reserved. This is a demo store only. Orders made WILL NOT be processed."
     }
   ],
+  "checkout_header.link.assistive_msg.cart": [
```

Comment:
![image](https://github.com/SalesforceCommerceCloud/pwa-kit/assets/52219283/5aa8eb3d-a1a8-4a41-8760-c8895b4af2c3)
Missing some translation for en-GB

Reply:
It seems like a common thing whenever we change/update translations. Can we add a command to include extracting to en-GB.json to avoid this mistake? 
```
node ./scripts/translations/extract-default-messages.js en-US && node ./scripts/translations/extract-default-messages.js en-GB
```

Reply:
@alexvuong I've duplicated the changes for en-GB too.

Yeah, I see how this can be a common mistake... while you develop for the monorepo. I think the better place to fix it would be within our extract-default-messages script, rather than in the npm script itself. Because whatever we put in the npm scripts would get exported into the generated project (which is not multi-locales).

The extract-default-messages script could have this additional logic:
```js
if (monorepo) {
  // duplicate en-us translation and replace the existing en-gb
} 
```

Reply:
Can you create a ticket for it please? It is a low hanging fruit but it is good to fix it

Reply:
Ok, created ticket W-14537667.